docs/conf.py

```diff
@@ -157,14 +157,16 @@ class CqlLexer(BashLexer):
 TAGS = []
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ['master']
+BRANCHES = ['branch-4.3', 'branch-4.4', 'branch-4.5']
 smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
 # Whitelist pattern for remotes (set to None to use local branches only)
 smv_remote_whitelist = r"^origin$"
 # Pattern for released versions
 smv_released_pattern = r'^tags/.*$'
 # Format for versioned output directories inside the build directory
 smv_outputdir_format = '{ref.name}'
+smv_latest_version = 'branch-4.4'
```

Comment:
``branch-4.5`` seems to be newer than ``branch-4.4``. Is still 4.4 the latest stable version?

Reply:
yes